internal/api/ui/login/static/resources/generate.go

```diff
@@ -3,4 +3,4 @@ package resources
 // TODO whitelabeling go:generate sass themes/scss/zitadel-alternative.scss themes/zitadel/css/zitadel.css
 // scss
 
-//go:generate sass themes/scss/zitadel.scss themes/zitadel/css/zitadel.css
+//go:generate sass --silence-deprecation global-builtin,import themes/scss/zitadel.scss themes/zitadel/css/zitadel.css
```

Comment:
Is silencing deprecation warnings really a good idea? If the functionality is removed in a future release of sass it will completely break. IMO somebody should just investigate and upgrade whatever is being deprecated. (create an issue)

If the warnings that are printed don't break the generate command, I'll just let them be printed.

Reply:
I don't think we want to spend much time on the login v1 styles at this stage. However, you're right, it's better to have the warnings printed for visibility. So I'll remove the --silence flag.